src/Hosting/Server.IntegrationTesting/src/Deployers/RemoteWindowsDeployer/RemoteWindowsDeployer.cs

```diff
@@ -74,10 +74,7 @@ public override async Task<DeploymentResult> DeployAsync()
     {
         using (Logger.BeginScope("Deploy"))
         {
-            if (_isDisposed)
-            {
-                throw new ObjectDisposedException("This instance of deployer has already been disposed.");
```

Comment:
note we'll get some delta here in the message; not necessarily a problem, just perhaps warrants note in the PR top

Reply:
Good point, there were a couple of odd throw disposed exceptions that I meant to revisit.

The problem is the initial `ObjectDisposedException` call here is wrong. The first parameter is supposed to be the object name. Fixed.